### PR TITLE
Remove filters from pages no longer tied to all roles

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Removed filters from pages that no longer need them.

## Rationale

These pages used to be tabs that were accessible via the all roles page. The design of that page has changed and no longer includes these tabs, it doesn't make sense to show the filters on these pages anymore. 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- Visit the following pages:
  - https://canonical-com-711.demos.haus/careers/diversity/identity
  - https://canonical-com-711.demos.haus/careers/diversity
  - https://canonical-com-711.demos.haus/careers/ethics
  - https://canonical-com-711.demos.haus/careers/lifestyle
  - https://canonical-com-711.demos.haus/careers/progression
  - https://canonical-com-711.demos.haus/travel
- See that department and location filters have been removed 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-823

### CODE REVIEW

**Comment by marinacastejon** [comment]
FYI these pages should not go live before Gloria has finished reviewing all the content. To be confirmed by @anthonydillon

**Comment by anthonydillon** [comment]
This is using the current content. Do you know when the review of the new content is planned for? @marinacastejon

**Comment by marinacastejon** [comment]
Hi @anthonydillon , here is the [JIRA issue](https://warthogs.atlassian.net/browse/CSTRAT-168) in the board of the Content team (no info about expected delivery date).
-
For the record, it was decided in stand-up to keep these pages live with the current content.

**Comment by anthonydillon** [comment]
@mtruj013 are there any links to these pages in the current build?

**Comment by anthonydillon** [comment]
> Hi @anthonydillon , here is the [JIRA issue](https://warthogs.atlassian.net/browse/CSTRAT-168) in the board of the Content team (no info about expected delivery date).
> 
> For the record, it was decided in stand-up to keep these pages live with the current content.

@marinacastejon If you would like to hold off on the project until we have signed off content. It's your call.

**Comment by mtruj013** [comment]
@ant you mean in the department pages build? If so, no

**Comment by anthonydillon** [comment]
@marinacastejon where should these pages link from? Changes look good but would be worth linking to them. Seems like the experience has regressed.

**Review by anthonydillon — APPROVED** [review]
Technically this change is good :+1: nice and atomic change.

**Comment by marinacastejon** [comment]
What part of the experience has regressed in your opinion @anthonydillon ? 
The new sitemap and meganav (as agreed in Prague with our stakeholders) was done in [this issue](https://warthogs.atlassian.net/browse/WD-592)
Let me know if anything is not clear and if you want us to change anything in the middle iterations. 
FYI @mtruj013

**Comment by anthonydillon** [comment]
Just want to ensure we cover the right links to the right places as we go. These pages are currently not linked to from anywhere.

**Comment by marinacastejon** [comment]
Indeed. It was a problem I raised in the past with going live with intermediate versions. I'll schedule a meeting so that we can re-discuss this.

**Comment by mtruj013** [comment]
@anthonydillon @marinacastejon I did a bit of digging and figured out where people are reaching these pages from and it turns out it's from the ["Thank you" page](https://github.com/canonical/canonical.com/blob/5d4e9676ceeb4abb5565961dbdbe477e7996a08f/templates/careers/thank-you.html#L19) the candidate sees after submitting their application. Just to give my 2 cents, I do not think we should be linking to these pages at all for now, we should add a redirect to keep them hidden instead. They haven't gone through design review, the content presumable isn't our best (hence the need for a re-write, I assume), and right now they don't even follow the flow of the site map. I don't think incomplete work is something we should be showcasing to our candidates. It's obviously up to you guys though, let me know whatever you decide.

**Comment by marinacastejon** [comment]
I agree with @mtruj013

**Pull request merged**
